pkg/apiserver/builder/helper.go

```diff
@@ -25,6 +26,15 @@ import (
 	"github.com/grafana/grafana/pkg/apiserver/endpoints/filters"
 )
 
+var pathRewriters = []filters.PathRewriter{
```

Comment:
@ryantxu something like this? the rewriter itself is added in https://github.com/grafana/grafana/pull/87456

Reply:
Yes! this works great!